novuhq/website
# feat: update navbar

**Describe what changes this pull request brings**

Update navbar, both desktop and mobile

**Steps to check:**

1. Open [preview](https://deploy-preview-304--novu-website.netlify.app)
2. Make sure that everything looks and works as expected

**TODO before merge:**

- [x] Add all links to new menu items
